package.json

```diff
@@ -12,9 +12,8 @@
     "@babel/runtime": ">=7"
   },
   "dependencies": {
-    "clean-webpack-plugin": "^3.0.0",
-    "unused-files-webpack-plugin": "^3.4.0",
```

Comment:
this feels like something devs should decide on for each repo

Reply:
I think we were keeping this one in to make sure everyone is keeping an eye on unused files -- I think we should bring it up in our Front end meeting (I suppose next week since everyone is out tomorrow) so people can voice their thoughts on it?

Reply:
added it back in until further notice